### PR TITLE
[Feature] GSSOC: Theme page

### LINKED ISSUE #125: [Feature] GSSoC : Themes page from Link-in-bio page
# Feature to issue #5

### Description
m

- Upon clicking view more themes in the end, user should be redirected to themes page.
- This page wouldn't be included in side-bar but will remain as a redirected page like the help page.

### Changes You need to make

Changes in following files

- /src/components/pages/Theme.tsx
- /src/assets/icons.tsx

### Checklist

- [x] Gone through the [rules and guidelines](https://github.com/oxiton-foundation/click-metrics/issues/24) of the project.
- [x] When you are creating PR, your PR should title follows the specified format: [Feature/Enhancement/Bug Fix]: Brief description of the change.
- [x] Attached OSSI or GSSoC in the heading of the Issue.
- [x] I have checked this issue is not already covered in another issue.

### Design

-  Carousel type theme cards which can be user customizable too
-  Tech Stack: React, Vite, TailwindCSS, TypeScript, NextUI/RadixUI libraries for enhanced UI
-  Story user info for chosen them-cards using MongoDB

### PR DESCRIPTION
**Pull request by lazybug19**
# Resolves #125 

## Changes:

- Carousel of Link-in-bio theme cards with template for linking major social media apps
- Enhanced UI of Theme Page and Link-in-Bio Page
- Made page compatible with different screen sizes

## Screenshot:
image

### CODE REVIEW

**Comment by Swarnendu0123** [comment]
Remove the `package-lock.json` from client and push a commit.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (Swarnendu0123) [inline package.json:1]: Delete this files also.

**Comment by Swarnendu0123** [comment]
You didn't delete the files I mentioned. Please delete that.

**Comment by lazybug19** [comment]
Deleted files:

- package-lock.json from root dir
- package.json from root dir
- package-lock.json from client

**Comment by Swarnendu0123** [comment]
Thanks:)

**Pull request merged**